app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self,
+                 nickname: str,
+                 favourite_dish: str,
+                 skill_level: int
+                 ) -> None:
```

Comment:
Place each argument including `self` on a new line here and in the other classes so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability